frontend/src/components/ItemGallery.js

```diff
@@ -0,0 +1,38 @@
+import ItemCard from './ItemCard'
+
+export default function ItemGallery({
+  List,
+  valueTitle,
+  reloadList,
+  editService,
+  removeService,
+}) {
+  if (valueTitle === 'Food') {
+    return List.map(Item => (
+      <ItemCard
+        valueTitle={valueTitle}
```

Comment:
```suggestion
export default function ItemGallery({
  list,
  valueTitle,
  reloadList,
  editService,
  removeService,
}) {
  if (valueTitle === 'Food') {
    return list.map(item => (
      <ItemCard
        valueTitle={valueTitle}
```
Capital letters for react components **only!**

Think about speaking names, is list just a list or is it the items of the gallery? Imho `items` would be a better name.